.travis.yml

```diff
@@ -4,9 +4,7 @@ script:
 node_js:
     - "4"
     - "6"
-    - "7"
```

Comment:
Maybe we should follow the same approach:
https://github.com/restify/node-restify/blob/master/.travis.yml#L4

Reply:
We can, but stable (node10) is currently broken, even in restify. Something changed in latest 10.x that broke us - need to look into it.